pkg/ruler/frontend_client.go

```diff
@@ -101,5 +101,9 @@ func (p *FrontendClient) InstantQuery(ctx context.Context, qs string, t time.Tim
 		level.Warn(log).Log("warnings", warning, "query", qs)
 	}
 
+	if len(infos) > 0 {
+		level.Warn(log).Log("infos", infos, "query", qs)
+	}
```

Comment:
I think we can remove this log as info is not warning at all. No need to log them